src/rust/cryptography-x509-validation/src/policy/mod.rs

```diff
@@ -234,15 +249,340 @@ impl<'a, B: CryptoOps> Policy<'a, B> {
                 .into_iter()
                 .cloned()
                 .collect(),
+            common_extension_policies: Vec::from([
+                // 5280 4.2.1.8: Subject Directory Attributes
+                ExtensionPolicy::maybe_present(
+                    SUBJECT_DIRECTORY_ATTRIBUTES_OID,
+                    Criticality::NonCritical,
+                    None,
+                ),
+                // 5280 4.2.2.1: Authority Information Access
+                ExtensionPolicy::maybe_present(
+                    AUTHORITY_INFORMATION_ACCESS_OID,
+                    Criticality::NonCritical,
+                    Some(common::authority_information_access),
+                ),
+            ]),
+            ca_extension_policies: Vec::from([
+                // 5280 4.2.1.1: Authority Key Identifier
+                ExtensionPolicy::maybe_present(
+                    AUTHORITY_KEY_IDENTIFIER_OID,
+                    Criticality::NonCritical,
+                    Some(ca::authority_key_identifier),
+                ),
+                // 5280 4.2.1.2: Subject Key Identifier
+                ExtensionPolicy::present(
```

Comment:
Sounds good, will tweak.

Just FYI -- SecureTrust *does* appear to have a SKI, at least for the one I found in `certifi`'s CA bundle:

```
-----BEGIN CERTIFICATE-----
MIIDuDCCAqCgAwIBAgIQDPCOXAgWpa1Cf/DrJxhZ0DANBgkqhkiG9w0BAQUFADBI
MQswCQYDVQQGEwJVUzEgMB4GA1UEChMXU2VjdXJlVHJ1c3QgQ29ycG9yYXRpb24x
FzAVBgNVBAMTDlNlY3VyZVRydXN0IENBMB4XDTA2MTEwNzE5MzExOFoXDTI5MTIz
MTE5NDA1NVowSDELMAkGA1UEBhMCVVMxIDAeBgNVBAoTF1NlY3VyZVRydXN0IENv
cnBvcmF0aW9uMRcwFQYDVQQDEw5TZWN1cmVUcnVzdCBDQTCCASIwDQYJKoZIhvcN
AQEBBQADggEPADCCAQoCggEBAKukgeWVzfX2FI7CT8rU4niVWJxB4Q2ZQCQXOZEz
Zum+4YOvYlyJ0fwkW2Gz4BERQRwdbvC4u/jep4G6pkjGnx29vo6pQT64lO0pGtSO
0gMdA+9tDWccV9cGrcrI9f4Or2YlSASWC12juhbDCE/RRvgUXPLIXgGZbf2IzIao
wW8xQmxSPmjL8xk037uHGFaAJsTQ3MBv396gwpEWoGQRS0S8Hvbn+mPeZqx2pHGj
7DaUaHp3pLHnDi+BeuK1cobvomuL8A/b01k/unK8RCSc43Oz969XL0Imnal0ugBS
8kvNU3xHCzaFDmapCJcWNFfBZveA4+1wVMeT4C4oFVmHursCAwEAAaOBnTCBmjAT
BgkrBgEEAYI3FAIEBh4EAEMAQTALBgNVHQ8EBAMCAYYwDwYDVR0TAQH/BAUwAwEB
/zAdBgNVHQ4EFgQUQjK2FvoE/f5dS3rD/fdMQB1aQ68wNAYDVR0fBC0wKzApoCeg
JYYjaHR0cDovL2NybC5zZWN1cmV0cnVzdC5jb20vU1RDQS5jcmwwEAYJKwYBBAGC
NxUBBAMCAQAwDQYJKoZIhvcNAQEFBQADggEBADDtT0rhWDpSclu1pqNlGKa7UTt3
6Z3q059c4EVlew3KW+JwULKUBRSuSceNQQcSc5R+DCMh/bwQf2AQWnL1mA6s7Ll/
3XpvXdMc9P+IBWlCqQVxyLesJugutIxq/3HcuLHfmbx8IVQr5Fiiu1cprp6poxkm
D5kuCLDv/WnPmRoJjeOnnyvJNjR7JLN4TJUXpAYmHrZkUjZfYGfZnMUFdAvnZyPS
CPyI6a6Lf+Ew9Dd+/cYy2i2eRDAwbO4H3tI0/NL/QPZL9GZGBlSm8jIKYyYwa5vR
3ItHuuG51WLQoqD0ZwV4KWMabwTW+MZMo5qxN7SN5ShLHZ4swrhovO0C7jE=
-----END CERTIFICATE-----
```

(VeriSign Class 3 doesn't have an SKI, but it also isn't in the `certifi` bundle. I'm guessing it's in the Oracle root program or similar, since the Oracle SSL CA is issued from it?)

Reply:
idk, ask @reaperhulk

Reply:
c6d502e5a